.github/workflows/ci.yml

```diff
@@ -22,7 +22,7 @@ jobs:
                 # - /libqtile/scripts/check.py mypy argument
                 # If adding new python versions, consider also updating
                 # python version in .readthedocs.yaml
-                python-version: [pypy-3.7, 3.7, 3.8, 3.9, '3.10']
+                python-version: [pypy-3.9, 3.8, 3.9, '3.10', '3.11']
```

Comment:
Per the comment, should we bump the version in .readthedocs.yaml to 3.11 too?

Reply:
Yes. We can do that. It may break a couple of the screenshots while dbus-next isn't fixed, though.

Reply:
I suppose for the docs it doesn't matter which version is used and the result would be the same? If so then yeah let's leave it for now